---
layout: post
title: KAPA Stranded mRNA Seq Prep
date: '2024-11-08'
categories: Protocols
tags: [Crassostrea virginica, oyster, larval, mRNA, RNA, RNAseq, Larval-CADO, Vibrio, Library Prep]
projects: [ASMFC-EOBC]
---


## KAPA Stranded mRNA Seq Prep of Larval Oyster Samples 

Using [KAPA Stranded mRNA Seq Kit with mRNA Capture Beads](https://sequencing.roche.com/us/en/products/group/kapa-stranded-rna-seq-kits.html) 

Following the [KAPA mRNA HyperPrep Kit Protocol](https://genome.med.harvard.edu/documents/libraryPrep/KAPAmRNAHyperPrepProtocol.pdf) using half volumes. 

**Time to Completion:**

### Sample Preparation

Prior to bead capture, replicate RNA samples for each treatment were pooled (ex. ARC2_CB1-3 were polled to ARC2_Con). Experimental trials were kept separate. This resulted in a con and stress pool for 7 larval CADO trials (ARC2, NEH1-3, MV1-3) and 4 treatment pools for the MV3 Vibrio challenge. Resulting in a total of **18pools** for mRNA library prep.

For mRNA Seq prep, the desired input was 1200ng of RNA concentrated in 25uL (half-reaction)

All calculations and notes for pooling RNA samples can be accessed on the [Guidry_mRNA_prep]() google sheet.

#### Pooling samples  
Since some samples were extracted ~9 months ago, RNA concentrations will be requantified with a qubit assay prior to pooling.
- Pull samples out of -80 freezer and thaw on ice 
- Perform [RNA Broad Range qubit assay]()
- some samples may need to be diluted and re-analyzed
- Using new qubit readings, pool samples as described above 
- Pipette pooled samples to mix and spin down 
- re-freeze for later bead capture or move on to bead capture


### Reagents 

KAPA Stranded mRNA-Seq Kit (KAPA #KK8420). This kit includes all the enzymes and buffers required for cDNA library preparation from  isolation of poly(A)-tailed RNA. Kits include reagents for RNA fragmentation, 1st strand cDNA synthesis and 2nd strand synthesis/marking, and cDNA library preparation, including A-tailing, ligation and library amplification. 

### Steps in Library construction:

* mRNA capture using magnetic oligo-dT beads
* Fragmentation using heat and magnesium
* 1st Strand cDNA Synthesis using random priming
* 2nd Strand cDNA Synthesis and marking, which converts the cDNA:RNA hybrid to double-stranded cDNA (dscDNA) and incorporates dUTP in the second cDNA strand
* A-tailing to add dAMP to the 3′-ends of the dscDNA library fragments
* Adapter ligation, where dsDNA adapters with 3′-dTMP overhangs are ligated to A-tailed library insert fragments
	* **NOTE** Here, we insert custom adapters. See below.
* Library amplification to amplify library fragments carrying appropriate adapter sequences at both ends using high-fidelity, low-bias PCR; the strand marked with dUTP is not amplified.

### Tubes sets needed
- Bead Capture
- 1st & 2nd strand synthesis
- Adapter Ligation 
- Qubit tubes
- Tape station tubes

### Mixes to make
- 1x FPE
- 1st strand cDNA synthesis mix
- 2nd strand cDNA synthesis mix
- 80% EtOH
- A-Tailing mix
- Adapter ligation mix
- SAIIv2 adapter mix
- 10mM Tris HCl pH 8.0


### Equipment 
* Magnetic stand and compatible tubes or striptubes 
* Thermocycler
* SPRI purification beads (KAPA Pure Beads or AmpureXP)

### Custom Oligos needed to make adapters:
See [EecSeq protocol](https://github.com/jpuritz/EecSeq/blob/master/Protocol.md)

### Bead wash & mRNA Bead Capture
- Pull bead kit out of fridge and warm to room temperature
- Gently invert bead bottle and pipette to suspend beads 
- Calculate number of beads needed for samples
    - 26.25 μL x # of samples 
    - *For 10 samples, 263 μL of beads are needed*
- Pipette the calculated volume of beads (263 μL) into a 1.5 mL tube and put tube on large magnet stand 
- Remove liquid when clear, being careful not to disturb the beads
- Add the same volume (263 μL) of Bead Binding Buffer and resuspend beads off of magnet
- Put tube back on magnet, remove liquid when clear (as much as possible)
- Add the same volume (263 μL) of Bead Binding Buffer and resuspend beads off of magnet
- Put tube back on magnet, remove liquid when clear (as much as possible)
- Add the same volume (263 μL) of Bead Binding Buffer and resuspend beads off of magnet
- Add 25 μL of the resuspended mRNA beads to each RNA sample and pipette to mix
- Thermocycler 1st mRNA Capture program 
- After thermocycler program, place tubes on magnet and discard liquid when clear 
- Remove tubes from magnet and resuspend beads in 100 μL of Bead Wash Buffer
- Place tubes on magnet and remove liquid when clear 
- Remove tubes from magnet and resupend beads in 25 μL of RNAse-free water
- Thermocycler 2nd mRNA Capture program
- After thermocycler program, add 25 μL of Bead Binding Buffer and pipette to mix
- Incubate tubes at 20 degrees C for 5 minutes in thermocycler (program exists)
- Make 1X Fragment, Prime, and Elute (FPE) Buffer on ice 
    - Nuclease-free water: 5.5 μL x # of samples
    - 2X FPE Buffer: 5.5 μL x # of samples
- After incubation, place tubes on magnet and discard liquid when clear
- Remove tubes from magnet and resuspend beads in 100 μL of Bead Wash Buffer
- Place tubes on magnet and discard liquid when clear 
- Remove tubes from magnet and resuspend beads in 11 μL of 1X FPE Buffer 
- ***SAFE STOPPING POINT:*** Resuspended beads with captured mRNA may be stored at 4C for up to 24 hours. Do not freeze the samples as this will damage the beads. When ready, proceed to step below.
- Thermocycler RNA Fragmentation program (7 minutes at 94 degrees C)

### First and Second Strand Synthesis

- Make 1st Strand cDNA Synthesis mix on ice:
    - 1st Strand Synthesis Buffer: 5.5 μL x # of samples
    - KAPA script: 0.5 μL x # of samples 
- **After thermocycler program, *IMMEDIATELY* put tubes on magnet**
- Transfer 10 μL of clear liquid to new PCR tubes on ice
- Add 5 μL of 1st Strand Synthesis mix to each sample and pipette to mix (still on ice)
- Thermocycler 1st Strand Synthesis program (~40 minutes)
- Make 2nd Strand Synthesis mix on ice:
    - 2nd Strand Marking Buffer: 15.5 μL x # of samples
    - 2nd Strand Enzyme: 1 μL x # of samples
- After thermocycler program, place tubes on ice
- Add 15 μL of 2nd Strand Synthesis mix to each sample and pipette to mix 
- Thermocycler 2nd Strand Synthesis program (~1 hour)
-  Take KAPA Pure Beads out of fridge to warm to room temperature (light sensitive)
- Make 80% EtOH (with molecular pure water)
- Take tubes out of thermocycler and add 54 μL of KAPA Pure Beads to each tube and pipette to mix
- Incubate on shaker for 15 minutes at 300 rpm and at room temperature
- Make A-Tailing Immediately mix on ice:
    - Nuclease-free water: 12 μL x # of samples
    - 10X A-Tailing Buffer: 1.5 μL x # of samples
    - A-Tailing Enzyme: 1.5 μL x # of samples
- After 15 minutes on shaker, perform normal bead clean steps
    - Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
    - Carefully remove and discard 74 μl of supernatant.
    - *Keeping the plate/tube on the magnet*, add 200 μl of 80% ethanol.
    - Incubate the plate/tube at room temperature for ≥30 sec.
    - Carefully remove and discard the ethanol.
    - *Keeping the plate/tube on the magnet*, add 200 μl of 80% ethanol.
    - Incubate the plate/tube at room temperature for ≥30 sec.
    - Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
    - Dry the beads at room temperature, until all of the ethanol has evaporated. 
        - **Caution: over-drying the beads may result in dramatic yield loss.**
- *IMMEDIATELY* resuspend beads in 15 μL of A-Tailing Immediately mix **OR** follow safe stopping point intructions below
- Thermocycler A-Tailing program (~1 hour) then proceed to **Adapter Ligation**
- ***SAFE STOPPING POINT:*** Resuspend the beads in 7.5 μl 1X A-Tailing Buffer (see table above), cover the reaction and store at 4C for up to 24 hours. Do not freeze the samples as this will damage the AMPure® XP® beads. When ready, proceed to A-Tailing after Safe Stopping Point

#### A-Tailing after safe stopping point (if not done immediately)
- Make the A-Tailing mix on ice:
    - Nuclease-free water: 5.25 μL x # of samples
    - A-tailing buffer: 0.75μL x # of samples
    - A-tailing enzyme: 1.5μL x # of samples
- Add 7.5μL of the A-Tailing after safe stopping point mix and pipette to mix
- Thermocycler A-Tailing program (~1 hour) then proceed to **Adapter Ligation**



### Adapter Ligation
***This will be where we insert the custom adapters that are barcoded with RE sites***

- Make Adapter Ligation mix on ice:
    - Nuclease-free water: 8 μL x # of samples
    - Ligation Buffer: 7 μL x # of samples
    - DNA Ligase: 2.5 μL x # of samples (pull out 30 minutes before use)
- Make 700 nM working stock of SAIIv2 adapter - we have a 1.5 μM stock in the freezer
    - Total volume needed of diluted adapter: 2.5 μL x # of samples
    - Volume of SAIIv2 adapter (700nM/1500 nM) x 2.5 μL x # of samples
    - Fill remaining volume with nuclease-free water
- After thermocycler program, add 17.5 μL of Ligation mix and 2.5 μL of SAIIv2 (700  nM) to each sample on ice and pipette to mix
- Place tubes on shaker at 300 rpm at room temperature for 30 minutes
- After 30 minute incubation, add 35 μL of room temperature PEG to each sample and pipette to mix
- Perform normal bead clean steps
    - Thoroughly resuspend the beads by pipetting up and down multiple times.
    - Incubate the plate/tube at room temperature for 5 – 15 min to allow the DNA to bind to the beads.
    - Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
    - Carefully remove and discard 74 μl of supernatant.
    - *Keeping the plate/tube on the magnet*, add 200 μl of 80% ethanol.
    - Incubate the plate/tube at room temperature for ≥30 sec.
    - Carefully remove and discard the ethanol.
    - *Keeping the plate/tube on the magnet*, add 200 μl of 80% ethanol.
    - Incubate the plate/tube at room temperature for ≥30 sec.
    - Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
    - Dry the beads at room temperature, until all of the ethanol has evaporated. 
        - **Caution: over-drying the beads may result in dramatic yield loss.**
- Resuspend beads in 25 μL of 10 mM Tris HCl pH 8.0
- ***SAFE STOPPING POINT:*** The solution with resuspended beads can be stored at 4 °C for up to 24 hours. Do not freeze the beads, as this can result in dramatic loss of DNA. When ready, proceed to 2nd Post-Ligation Cleanup
- Add 25 μL of room temperature PEG to each sample and pipette to mix
- Perform normal bead clean steps 
    - Thoroughly resuspend the beads by pipetting up and down multiple times.
    - Incubate the plate/tube at room temperature for 5 – 15 min to allow the DNA to bind to the beads.
    - Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
    - Carefully remove and discard 74 μl of supernatant.
    - *Keeping the plate/tube on the magnet*, add 200 μl of 80% ethanol.
    - Incubate the plate/tube at room temperature for ≥30 sec.
    - Carefully remove and discard the ethanol.
    - *Keeping the plate/tube on the magnet*, add 200 μl of 80% ethanol.
    - Incubate the plate/tube at room temperature for ≥30 sec.
    - Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
    - Dry the beads at room temperature, until all of the ethanol has evaporated. 
        - **Caution: over-drying the beads may result in dramatic yield loss.**
- Resuspend beads in 11 μL of 10 mM Tris HCl pH 8.0
- Transfer 10 μL of sample to new PCR strip tubes
- ***SAFE STOPPING POINT:*** The purified, adapter-ligated library DNA may be stored at 4 °C for up to 1 week, or frozen at -20 °C for up to 1 month. When ready, proceed to Library Amplification

### Library Amplification

- Set up amplification with KAPA HiFi HotStart Ready Mix (HSRM) and index primer pairs
    - We use custom index primers 
    - *recommended that a 20 μM stock is used to maximize amplification* 
    - Index primer pair combinations were chosen to multiplex cDNA libraries in 3 probe pools (separate pools for each Block)
    - Specific index primers used for each sample can be accessed [here]() under *Pooled_Samples* tab
- Combine 12.5 μL HSRM and 1.25 μL of each index primer into PCR tubes on ice
- Combine the 10 μL of adapter-ligated cDNA sample with 15 μL of the appropriate library amplification mix 
- Vortex and spin down
- Thermocycler PCR 12 program (~30 minutes)
- After PCR program, perform 1X bead clean 
    - Add 25 μL of KAPA Pure Beads
    - Thoroughly resuspend the beads by pipetting up and down multiple times.
    - Incubate the plate/tube at room temperature for 5 – 15 min to allow the DNA to bind to the beads.
    - Place the plate/tube on a magnet to capture the beads. Incubate until the liquid is clear.
    - Carefully remove and discard 74 μl of supernatant.
    - *Keeping the plate/tube on the magnet*, add 200 μl of 80% ethanol.
    - Incubate the plate/tube at room temperature for ≥30 sec.
    - Carefully remove and discard the ethanol.
    - *Keeping the plate/tube on the magnet*, add 200 μl of 80% ethanol.
    - Incubate the plate/tube at room temperature for ≥30 sec.
    - Carefully remove and discard the ethanol. Try to remove all residual ethanol without disturbing the beads.
    - Dry the beads at room temperature, until all of the ethanol has evaporated. 
        - **Caution: over-drying the beads may result in dramatic yield loss.**
    - Resuspend and elute beads in 22 μL of 10 mM Tris HCl pH 8.0
- Transfer liquid to new PCR tubes and store at -20C


### Quality Control Check 

**Qubit**
- cDNA library concentrations are checked using the Biotium [dsDNA Broad Range](https://biotium.com/wp-content/uploads/2017/12/PI-31069.pdf) quantitation kit
- Final cDNA library concentrations (ng/μL) can be accessed [here]() under *Pooled_Samples* tab

**TapeStation**
- cDNA libraries are checked using the [D5000 ScreenTape Assay](https://www.agilent.com/cs/library/usermanuals/public/ScreenTape_D5000_QG.pdf)
- Sample TapeStation report can be accessed [here]()
    - Libraries should be around 300 bp 